src/charm.py

```diff
@@ -179,6 +182,29 @@ def _on_get_initial_password_action(self, event):
             logger.error("Action get-initial-password failed. Replica consensus not exists")
             event.fail("Default admin password has not been generated yet.")
 
+    def _on_rotate_wordpress_secrets_action(self, event):
```

Comment:
We have a standard for including type hints for functions, do you think it is applicable here? https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#type-hints